sycl/test/basic_tests/image_accessor_types.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: not %clangxx -fsyntax-only -std=c++17 %s -I %sycl_include/sycl 2>&1 | FileCheck %s
+// RUN: %clangxx -fsyntax-only %fsycl-host-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s
```

Comment:
we are losing checks for the compilation of device-only code like Kosnt mentioned for ESIMD tests.

Reply:
I'm not exactly that this is needed here, but anyway returned host compilation back in 118ec382ffcb5d04ae1bf317e705024f875591bd